pkg/kubectl/cmd/util/openapi/validation/validation.go

```diff
@@ -64,7 +64,7 @@ func (v *SchemaValidation) validateList(object interface{}) []error {
 	errs := []error{}
 	for _, item := range fields["items"].([]interface{}) {
 		if gvk, err := getObjectKind(item); err != nil {
-			errs = append(errs, err)
+			errs = append(errs, utilerrors.NewAggregate(err))
```

Comment:
I forgot the most important: This is appending a nested list to the list.

It should read:
```
errs = append(errs, err...) // And maybe change the of errs?
```

Reply:
@apelisse I didn't know about variadic so thanks much for helping me educate here. I made changes and tested, it works :-)